Palaso/Settings/CrossPlatformSettingsProvider.cs

```diff
@@ -14,17 +14,27 @@ namespace Palaso.Settings
 	/// </summary>
 	public class CrossPlatformSettingsProvider : SettingsProvider, IApplicationSettingsProvider
 	{
+		//Public only for unit testing. I can't get InternalsVisibleTo to work, possibly because of strong naming.
+		public const string UserConfigFileName = "user.config";
```

Comment:
The other option is to make them internal and allow the test project to access internal stuff of the main assembly.